src/lib/process/index.ts

```diff
@@ -36,28 +36,60 @@ export const loadProcesses = async ({
   data,
   options,
   dryRun,
+  verbose,
 }: {
   data: string
   options?: Polkadot.Options
   dryRun?: boolean
+  verbose?: boolean
 }): Promise<Process.Response> => {
   const res: Process.Response = {}
   const processes: Process.CLIParsed = JSON.parse(data)
   // TODO more elegant promise.series way.
   for (let i = 0; i < processes.length; i++) {
     const { name, version, program } = processes[i]
-    res[name] = await createProcess(name, version, program, dryRun, options)
+    res[name] = await createProcess(name, version, program, dryRun, options, verbose)
   }
 
   return res
 }
 
+export const listProcess = async (
```

Comment:
This is being picky but the name of this function here feels incorrect. We're not listing processes. We're transforming the processes for output